client/src/main/java/org/schlunzis/kurtama/client/fx/controller/MainMenuController.java

```diff
@@ -56,6 +64,30 @@ protected void updateItem(LobbyInfo lobbyInfo, boolean empty) {
                 joinLobby();
         });
         joinLobbyButton.setDisable(true);
+
+        // whenever there is a change in the lobby list, update the auto-completion
+        AtomicReference<AutoCompletionBinding<String>> bindingRef = new AtomicReference<>();
```

Comment:
Using this `AtomicReference` was the first somewhat reasonable approach I could find, to tackle the problem with effectively final variables in lambdas. If you know something better, or even better, a best practice, please let me know.